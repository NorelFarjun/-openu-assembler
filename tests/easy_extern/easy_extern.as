lea LIST, r1
.extern LIST
