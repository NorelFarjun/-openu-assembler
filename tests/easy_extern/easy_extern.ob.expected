   3 0
0100 20504
0101 00001
0102 00014
